CODING_STYLE.md

```diff
@@ -0,0 +1,73 @@
+# Hoodie Coding Style Guidelines
+
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more guidelines on
+contributing to Hoodie.
+
+Hoodie uses the [Standard](https://github.com/feross/standard) JavaScript
+coding style.
+
+This file explains coding-style considerations that are beyond the syntax check
+of *Standard*.
+
+There are three sections:
+
+- *General*: coding styles that are applicable to all JavaScript code.
+- *Client*: coding styles that are only applicable to in-browser code.
+- *Server*: coding styles that are only applicable in server code.
+
+*Note: Client and Server coding styles can be contradicting, make sure to read
+these carefully*.
+
+
+## General
+
+Prefer [lodash](https://lodash.com) over [Underscore](http://underscorejs.org "Underscore.js").
```

Comment:
I don’t think this is worth mentioning any more. The [lodash / underscore merge](https://github.com/jashkenas/underscore/issues/2182) kinda sorta happened with lodash v4